src/styles/wikia-sharing.plugin.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../node_modules/design-system/dist/scss/wds-variables/index';
+
+.wikia-jw-sharing {
+	background-color: $wds-color-black;
+	border-radius: 4px;
+	bottom: 44px;
+	padding: 0 12px 24px;
+	pointer-events: auto;
+	position: absolute;
+	right: 12px;
+
+	h3 {
+		color: $wds-color-white;
+		display: flex;
+		font-size: 14px;
+		justify-content: center;
+		width: 100%;
+	}
+
+	&__plugin {
+		display: none;
+		height: 100%;
+		left: 0;
+		position: absolute;
+		top: 0;
+		width: 100%;
+		z-index: 3;
+	}
+
+	&__list {
+		color: $wds-color-white;
+		padding: 0;
+
+		li {
+			align-items: center;
+			cursor: pointer;
+			display: flex;
+			justify-content: space-between;
+
+			.jw-svg-icon {
+				margin: 0 17px 0 10px;
+			}
+		}
+	}
+
+	// keep controlbar and play icon on mobile open when sharing popover is open
+	&-open {
+		.jw-display, .jw-controlbar {
+			opacity: 1 !important;
+			visibility: visible !important;
```

Comment:
do we really have to use `!important` here?  😥

Reply:
Yes :) Talked offline.